tutorials/io/mergeFiles.py

```diff
@@ -0,0 +1,52 @@
+# \file
+# \ingroup tutorial_io
+# \notebook -nodraw
+# Illustrates how to merge two files using TFileMerger from Python.
+# \author Giacomo Parolini
+
+import ROOT
+import os
+
+def MergeFiles(files_to_cleanup):
+   # NOTE: when the TFileMerger is used in a `with` statement, it will automatically
+   # close its output file when going out of scope.
+   with ROOT.TFileMerger(False) as fm:
+      fm.OutputFile("merged.root")
+      files_to_cleanup.append(fm.GetOutputFile().GetName())
+      for i in range(0, nfiles):
+         fm.AddFile(f"tomerge{i}.root")
+
+      # New merging flags must be bitwise OR-ed on top of the default ones
+      mode = ROOT.TFileMerger.kAll | ROOT.TFileMerger.kIncremental
```

Comment:
Since this is a tutorial it would be nice to point users to what mode of operation is being used, can we either explain it in a comment or direct the reader to the TFileMerger docs?